contracts/starknet/lib/voting_library.cairo

```diff
@@ -0,0 +1,1066 @@
+# SPDX-License-Identifier: MIT
+
+%lang starknet
+
+from starkware.starknet.common.syscalls import get_caller_address, get_block_timestamp
+from starkware.cairo.common.cairo_builtins import HashBuiltin, SignatureBuiltin, BitwiseBuiltin
+from starkware.cairo.common.alloc import alloc
+from starkware.cairo.common.uint256 import Uint256, uint256_add, uint256_lt, uint256_le
+from starkware.cairo.common.bool import TRUE, FALSE
+from starkware.cairo.common.hash_state import hash_init, hash_update
+from starkware.cairo.common.math import (
+    assert_lt,
+    assert_le,
+    assert_nn,
+    assert_nn_le,
+    assert_not_zero,
+    assert_lt_felt,
+)
+
+from openzeppelin.access.ownable import Ownable
+from openzeppelin.account.library import Account, AccountCallArray, Account_current_nonce
+
+from contracts.starknet.Interfaces.IVotingStrategy import IVotingStrategy
+from contracts.starknet.Interfaces.IExecutionStrategy import IExecutionStrategy
+from contracts.starknet.lib.general_address import Address
+from contracts.starknet.lib.proposal import Proposal
+from contracts.starknet.lib.proposal_info import ProposalInfo
+from contracts.starknet.lib.vote import Vote
+from contracts.starknet.lib.choice import Choice
+from contracts.starknet.lib.proposal_outcome import ProposalOutcome
+from contracts.starknet.lib.hash_array import hash_array
+from contracts.starknet.lib.array2d import Immutable2DArray, construct_array2d, get_sub_array
+from contracts.starknet.lib.slot_key import get_slot_key
+
+#
+# Storage
+#
+
+@storage_var
+func Voting_voting_delay_store() -> (delay : felt):
```

Comment:
I followed the OZ extensibility [standard](https://github.com/OpenZeppelin/cairo-contracts/blob/main/docs/Extensibility.md): 
"Must prefix storage functions with the name of the namespace to prevent clashing with other libraries (e.g. ERC20_balances)"

Reply:
Yeah ok should've clicked on the link in the PR description ;)
It does give some weird variable names: `Voting_voting_delay_store`, `Voting_min_voting_duration_store`... it does look verbose. But if that's the standard, then we should definitely follow it :)